README.md

```diff
@@ -144,10 +144,10 @@ Now, define each of the following variables:
 - **Description:** Time to wait (in seconds) while polling for email sending status.
 - **Example:** `MEMBRANE_EMAIL_SEND_POLLER_WAIT_TIME=10`
 
-### MEMBRANE_EMAIL_SEND_HTLM_TEMPLATE
+### MEMBRANE_EMAIL_SEND_HTML_TEMPLATE
 
 - **Description:** Path to the HTML template used for outgoing emails.
-- **Example:** `MEMBRANE_EMAIL_SEND_HTLM_TEMPLATE=\<html>\<h1>{}\</h1>\</html>
+- **Example:** `MEMBRANE_EMAIL_SEND_HTML_TEMPLATE=\<html>\<h1>{}\</h1>\</html>
```

Comment:
are you sure you need the escape here? why not use " around the string?

Reply:
done

Reply:
Do you mean unit testing the email output to see if it is formatted based on the template?

Reply:
yes, link to test please